web/js/roomController.js

```diff
@@ -796,7 +796,7 @@ RecordingsController, ScreenShareController, FeedbackController, PhoneNumberCont
             };
           };
         loadAnnotations = LazyLoader.load([
-          'https://code.jquery.com/jquery-3.1.0.min.js',
+          '//ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js',
```

Comment:
Is this a safe place to load jquery for the phone call? 
it's a bit indirect. I wonder if the lazyloader is smart enough not to load scripts again and we can just put it in both places.

Reply:
Looks like the lazyloader doesn't check for dupes